pkg/kontainer-engine/drivers/eks/eks_driver.go

```diff
@@ -41,125 +41,180 @@ const (
 )
 
 var amiForRegionAndVersion = map[string]map[string]string{
+	"1.21": {
```

Comment:
Are you sure we want to support images for > 1.18 in 2.5?

Reply:
According to [this](https://github.com/rancher/rancher/issues/35295#issuecomment-984982454) comment, I believe so.